### PR TITLE
Naming

### PR DESCRIPTION
**Pull request by thattil**
renaming sls and detectors to modules, multis to detector

### CODE REVIEW

File: slsDetectorSoftware/src/DetectorImpl.cpp [hunk 1]
```diff
@@ -28,177 +28,173 @@
...
+    for (int i = 0; i < numModules; ++i) {
+        SharedMemory<sharedModule> moduleShm(detectorIndex, i);
+        moduleShm.RemoveSharedMemory();
     }
 }
 
 void DetectorImpl::freeSharedMemory() {
     zmqSocket.clear();
-    for (auto &d : detectors) {
+    for (auto &d : modules) {
```
Comment (erikfrojdh) [inline slsDetectorSoftware/src/DetectorImpl.cpp:78]: Maybe auto &m : modules? or module : modules?

File: slsDetectorSoftware/src/Module.h [hunk 2]
```diff
@@ -14,30 +14,30 @@
...
 
 namespace sls {
 
 /**
- * @short structure allocated in shared memory to store detector settings for
+ * @short structure allocated in shared memory to store Module settings for
  * IPC and cache
  */
-struct sharedSlsDetector {
+struct sharedModule {
```
Comment (erikfrojdh) [inline slsDetectorSoftware/src/Module.h:26]: Building of module tests fails, could you edit the tests?

**Review by erikfrojdh — CHANGES_REQUESTED** [review]
Fix broken build then merge

File: slsDetectorSoftware/src/DetectorImpl.cpp (hunk 1 above)
Comment (thattil) [reply slsDetectorSoftware/src/DetectorImpl.cpp]: changed all the d's to module

**Pull request merged**